<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>local_match</class>
 <widget class="QWidget" name="local_match">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>本地匹配</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>为本地歌曲文件匹配歌词</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="song_path_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="song_path_pushButton">
             <property name="text">
              <string>选择要遍历的文件夹</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>保存</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>歌词保存模式:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="save_mode_comboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>194</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>保存到歌曲文件夹的镜像文件夹</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>保存到歌曲文件夹</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>保存到指定文件夹</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="save_path_lineEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="save_path_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>选择文件夹路径</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>歌词文件名:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="lyrics_filename_mode_comboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>133</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>与设置中的格式相同</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>与歌曲文件名相同</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>歌词</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item alignment="Qt::AlignTop">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>歌词来源:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QCheckBox" name="qm_checkBox">
                      <property name="text">
                       <string>QQ音乐</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="ne_checkBox">
                      <property name="text">
                       <string>网易云音乐</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="kg_checkBox">
                      <property name="text">
                       <string>酷狗音乐</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>优先顺序:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="source_listWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>96</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::DragDrop</enum>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::MoveAction</enum>
                </property>
                <item>
                 <property name="text">
                  <string>QQ音乐</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>酷狗音乐</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>网易云音乐</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>歌词类型:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="original_checkBox">
                <property name="text">
                 <string>原文</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QCheckBox" name="translate_checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>译文</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QCheckBox" name="romanized_checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>罗马音</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start_cancel_pushButton">
           <property name="text">
            <string>开始匹配</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
